### PR TITLE
[actions] automatically answer missing windows SDK issues

### PR DESCRIPTION
**Pull request by autoantwort**
This kind of issue is reported over and over again and the answer is always the same: Install the windows SDK. So lets automate this.

### CODE REVIEW

**Review by Adela0814 — DISMISSED** [review]

**Comment by dg0yt** [comment]
I think this must be solved one step earlier: Clearly indicate in vcpkg tool's reporting that the SDK is missing, instead of suggesting to open an issue.

**Comment by autoantwort** [comment]
@dg0yt But I guess `vcpkg_cmake_configure` could detect that in the logs and suggest to install the windows SDK

File: .github/workflows/check_issues.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+              owner: context.repo.owner,
+              repo: context.repo.repo
+            };
+            let issue = await github.rest.issues.get(issue_query)
+            let reg = /RC Pass 1: command "rc .*" failed \(exit code 0\) with the following output:/;
+            if (reg.test(issue.data.body)){
+              await github.rest.issues.removeAllLabels(issue_query);
+              await github.rest.issues.setLabels({...issue_query, labels: ["category:question"]});
+              let body = "Thanks for posting this issue. Please make sure you have the following installed.\n" +
+                         "- Visual Studio English Language Pack.\n" +
```
Comment (BillyONeal) [inline .github/workflows/check_issues.yml:28]: I don't believe the en language pack should be necessary any longer? Maybe vcpkg_copy_pdbs still needs to be fixed...

Comment (autoantwort) [reply .github/workflows/check_issues.yml:28]: I have only copied what is suggested by your colleagues like [here](https://github.com/microsoft/vcpkg/issues/30534#issuecomment-1489801503)

Comment (autoantwort) [reply .github/workflows/check_issues.yml:28]: But I can remove it.

**Comment by BillyONeal** [comment]
> @Adela0814 Can you add the [ info:reviewed](https://github.com/microsoft/vcpkg/labels/info%3Areviewed) Pull Request changes follow basic guidelines label back?

No, because I'm merging it. Thanks!

File: .github/workflows/check_issues.yml
Comment (Powerm4KK) [inline .github/workflows/check_issues.yml:1]: /

**Review by Powerm4KK — COMMENTED** [review]
#30547

**Pull request merged**